frontend/src/components/App/Home/config.ts

```diff
@@ -0,0 +1,6 @@
+/** Allow selecting multiple clusters on home page. */
+export const MULTI_HOME_ENABLED = import.meta.env.REACT_APP_MULTI_HOME_ENABLED === 'true' || true;
+
+/** Show recent clusters on the home page */
+export const ENABLE_RECENT_CLUSTERS =
+  import.meta.env.REACT_APP_ENABLE_RECENT_CLUSTERS === 'true' || false;
```

Comment:
I think `|| false` here is also not needed

Reply:
done